Configuration/DataProcessing/python/Impl/hcalnzsEra_Run2_2018_pp_on_AA.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""
+_hcalnzsEra_Run2_2018_pp_on_AA_
+
+Scenario supporting 2018 heavyIon collisions
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Impl.hcalnzs import hcalnzs
+from Configuration.Eras.Era_Run2_2018_pp_on_AA_cff import Run2_2018_pp_on_AA
+
+class hcalnzsEra_Run2_2018_highBetaStar(hcalnzs):
+    def __init__(self):
+        hcalnzs.__init__(self)
+        self.recoSeq=':reconstruction_HcalNZS'
+        self.cbSc='pp'
+        self.addEI=True
```

Comment:
apparently the 3 lines above are superfluous, they somehow sneaked in through copy-paste from old scenarios

Reply:
removed in last commit